#!/usr/bin/env python
PACKAGE = "peek_thermal"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# - Original Heiko values
#gen.add("lower_H",  int_t,  0, "The lower H value", 0, 0, 255)
#gen.add("lower_S",  int_t,  0, "The lower S value", 0, 0, 255)
#gen.add("lower_V",  int_t,  0, "The lower V value", 88, 0, 255)
#gen.add("upper_H",  int_t,  0, "The upper H value", 95, 0, 255)
#gen.add("upper_S",  int_t,  0, "The upper S value", 137, 0, 255)
#gen.add("upper_V",  int_t,  0, "The upper V value", 255, 0, 255)

gen.add("lower_H",  int_t,  0, "The lower H value", 0, 0, 180)
gen.add("lower_S",  int_t,  0, "The lower S value", 0, 0, 255)
gen.add("lower_V",  int_t,  0, "The lower V value", 0, 0, 255)
gen.add("upper_H",  int_t,  0, "The upper H value", 119, 0, 180)
gen.add("upper_S",  int_t,  0, "The upper S value", 255, 0, 255)
gen.add("upper_V",  int_t,  0, "The upper V value", 249, 0, 255)


exit(gen.generate(PACKAGE, "peek_thermal", "hsv_parameters"))
